packages/ods-react/src/components/combobox/src/components/combobox/Combobox.tsx

```diff
@@ -0,0 +1,96 @@
+import type { ComboboxProp } from '../../context/useCombobox';
+import {
+  type ComboboxInputValueChangeDetails,
+  type ComboboxValueChangeDetails,
+  Combobox as VendorCombobox,
+  createListCollection,
+} from '@ark-ui/react/combobox';
+import { type FC, type JSX, forwardRef, useState } from 'react';
+import { ComboboxProvider } from '../../context/useCombobox';
+import { combobox } from '../../controller/combobox';
+
+const Combobox: FC<ComboboxProp> = forwardRef(({
+  allowCustomValue = true,
+  children,
+  className,
+  customOptionRenderer,
+  defaultValue,
+  disabled = false,
+  highlightResults = false,
+  items,
+  newElementLabel = 'Add ',
+  noResultLabel = 'No results found',
+  onValueChange,
+  readOnly = false,
+  value = [],
+  ...props
+}, ref): JSX.Element => {
+  const [inputValue, setInputValue] = useState('');
+  const [internalValue, setInternalValue] = useState<string[]>(defaultValue || []);
```

Comment:
is it really mandatory to store the value? it should be available in the ark context already (see https://zagjs.com/components/react/combobox#machine-context)